src/Illuminate/Collections/functions.php

```diff
@@ -17,11 +17,20 @@
      */
     function enum_value($value, $default = null)
     {
-        return transform($value, fn ($value) => match (true) {
+        if (function_exists('transform')) {
+            return transform($value, fn ($value) => match (true) {
+                $value instanceof \BackedEnum => $value->value,
+                $value instanceof \UnitEnum => $value->name,
+
+                default => $value,
+            }, $default ?? $value);
+        }
+
+        return match (true) {
             $value instanceof \BackedEnum => $value->value,
             $value instanceof \UnitEnum => $value->name,
 
             default => $value,
```

Comment:
Shouldn't the `default` arm be:

```php
    default => $value ?? $default,
```

Otherwise, the `$default` value is never used, when `transform()` isn't available

Reply:
I probably going to remove `$default` options completely. It only used once in `ModelInspector` class.

Reply:
on second thought, you suggest works fine but instead use `value($default)`. `value()` function available from `helpers.php` file in `illuminate/collections`